mdbook/src/notifications.md

```diff
@@ -1,36 +1,58 @@
+* When a value doesn't fall within any `meta.zones`, the server raises a default notification indicating a normal value state.
+* If the value transitions to or enters a zone, the server raises the defined notification.
+* If the value leaves all zones, the server sends a normal state value notification.
+* Notifications are raised by placing a notification object in the `vessels.self.notifications` tree.
 
-* The server (or device) should monitor the current value and compare it to the defined zones.
-* If a value enters an alarm zone, then a key is written to `vessels.self.notifications..`
-* If a value leaves an alarm zone, then the key is removed from `vessels.self.notifications..`
-* Alarms raised are monitored by an alarm process on the server, which takes appropriate action, sounding alarms, or
-  displaying messages.
-* Clients interested in alarms can subcribe to the `vessels.self.notifications...` tree in the usual way and be informed
-  of alarms in the same way as normal signalk keys.
-* When an alarms is removed, a delta should be sent to subscribers with the path and a null value.
+## Expected Implementation Behavior
+
+* The server (or device) should monitor the current value and compare it to the defined meta zones.
+* If the value changes zone, a key is written to `vessels.self.notifications..`.
+* If the value matches no zone, a key indicating normal state is written to `vessels.self.notifications..`.
+* Notifications raised are monitored by a notification process on the server, which takes appropriate action, such as sounding alarms or displaying messages.
+* Clients interested in alarms can subscribe to the `vessels.self.notifications...` tree in the usual way and be informed of notifications in the same way as normal Signal K keys.
+* When a notification is cleared, a delta should be sent to subscribers indicating the notification has returned to its normal state.
+
+## Severity and Presentation Definition
+
+* Notification must include a `state` property to specify its severity level.
+* Notification must include a `method` array property to determine if the notification should result in a sound, visual, both, or no presentation.
+
+| State | Description |
+|------------|--------|
+| normal     | Server generated - The normal value (default) |
+| nominal    | All systems OK |
+| alert      | Indicates a safe or normal condition which is brought to the operators attention to impart information |
+| warn       | Indicates a condition that requires immediate attention but not immediate action |
+| alarm      | Indicates a condition which is outside the specified acceptable range. Immediate action is required to prevent loss of life or equipment damage |
+| emergency  | The value indicates a life-threatening condition |
+
+## Resolving and silencing notifications
+* To silence a notification, the serve or clients, send delta notification replica without the `sound` item the method array.
```

Comment:
Actioning a notification is an operation of the notification manager. 
As this applies to implementation the expected operation should be outlined here.

Reply:
I thought about this yesterday, I was including Alarm Clearer plugin as a standard setup in this writing. What do you recommend?

Reply:
It would be nice to be able to explain how consumers should work the system.